src/enlyze/resampling.py

```diff
@@ -0,0 +1,31 @@
+import enlyze.models as user_models
+from enlyze.validators import validate_resampling_method_for_data_type
+
+
+def convert_to_variable_with_resampling_method(
+    variable: user_models.Variable, resampling_method: user_models.ResamplingMethod
+) -> user_models.VariableWithResamplingMethod:
+    """Convert a :class:`~enlyze.models.Variable` to a :class:`~enlyze.models.VariableWithResamplingMethod`.
+
+    :param variable: The variable to convert.
+    :type variable: :class:`~enlyze.models.Variable`
```

Comment:
Is there a reason why you specify the type here again? Sphinx usually can get it from the argument type annotations :thinking:

Reply:
Fixed.